src/atoms/Auth.ts

```
@@ -1,13 +1,79 @@
-import {atom} from 'jotai';
+import {atom, useAtom, useSetAtom} from 'jotai';
 
 import {atomForLocal} from '@/atoms/atomForLocal';
 import {User} from '@/generated-sources/openapi';
+import useAppComponents from '@/hooks/useAppComponents';
+import Logger from '@/logging/Logger';
+import { logDebug } from '@/utils/DebugUtils';
```

ESLint

[error] Replace `·logDebug·` with `logDebug`

(prettier/prettier)